File: src/net/minecraft/client/render/entity/CowEntityRenderer.cpp
#include "net/minecraft/client/render/entity/EntityRenderers.hpp"
#include "net/minecraft/client/entity/EntityClientRendererRegistration.hpp"
#include "net/minecraft/client/render/entity/model/CowEntityModel.hpp"
#include "net/minecraft/entity/passive/CowEntity.hpp"
namespace net::minecraft::entity::passive {
std::unique_ptr<::net::minecraft::client::render::entity::EntityRenderer> CowEntity::ClientRenderer::create() {
  using net::minecraft::client::render::entity::CowEntityRenderer;
  using net::minecraft::client::render::entity::model::CowEntityModel;
  return std::make_unique<CowEntityRenderer>(new CowEntityModel(), 0.7f);
}
} // namespace net::minecraft::entity::passive
namespace {
static ::net::minecraft::registry::RegisterEntityRenderer<::net::minecraft::entity::passive::CowEntity> autoRendererReg;
} // namespace
namespace net::minecraft::client::render::entity {
CowEntityRenderer::CowEntityRenderer(model::EntityModel* model, float shadowRadius)
    : LivingEntityRenderer(model, shadowRadius) {
}
} // namespace net::minecraft::client::render::entity

File: mcp/src/net/minecraft/client/render/entity/CowEntityRenderer.java
/*
 * Decompiled with CFR 0.1.1 (FabricMC 57d88659).
 * 
 * Could not load the following classes:
 *  net.fabricmc.api.EnvType
 *  net.fabricmc.api.Environment
 */
package net.minecraft.client.render.entity;

import net.fabricmc.api.EnvType;
import net.fabricmc.api.Environment;
import net.minecraft.client.render.entity.LivingEntityRenderer;
import net.minecraft.client.render.entity.model.EntityModel;
import net.minecraft.entity.passive.CowEntity;

@Environment(value=EnvType.CLIENT)
public class CowEntityRenderer
extends LivingEntityRenderer {
    public CowEntityRenderer(EntityModel arg, float f) {
        super(arg, f);
    }

    public void render(CowEntity arg, double d, double e, double f, float g, float h) {
        super.render(arg, d, e, f, g, h);
    }
}

